week1/react-exercises/src/components/Header.js

```diff
@@ -0,0 +1,9 @@
+const Header = ({ title }) => {
+  return (
+    <>
+      <h2>{title}</h2>
+    </>
+  );
+};
```

Comment:
You don't need a fragment when you're only returning one element

```suggestion
  return (
      <h2>{title}</h2>
  );
};
```